### PR TITLE
tkt-79239: Ensure UPS is not ONLINE at shutdown event

### PR DESCRIPTION
**Pull request by sonicaj**
This commit ensures UPS is not ONLINE when shutdown event is passed by upssched. There are cases where battery/charger issues can result in ups.status being 'OL LB' at the same time. This will ensure that we don't initiate a shutdown if ups is OL.
Ticket: #79239

### CODE REVIEW

File: src/middlewared/middlewared/plugins/ups.py [hunk 1]
```diff
@@ -178,11 +178,32 @@ def driver_choices(self):
...
+                fr'ups.status: (.*)',
+                '' if not stats_output else stats_output.decode()
+            )
+            if ups_status and 'ol' in ups_status[0].lower():
+                self.middleware.logger.debug(
+                    f'Shutdown not initiated as ups.status ({ups_status[0]}) indicates '
+                    f'{config["identifier"]} is ONLINE (OL).'
+                )
+            else:
+                syslog.syslog(syslog.LOG_INFO, 'upssched-cmd "issuing shutdown"')
```
Comment (themylogin) [inline src/middlewared/middlewared/plugins/ups.py]: @william-gr we do use syslog in middlewared, maybe we can start getting rid of it now?

File: src/middlewared/middlewared/plugins/ups.py [hunk 2]
```diff
@@ -178,11 +178,32 @@ def driver_choices(self):
...
+
         if notify_type.lower() == 'shutdown':
-            syslog.syslog(syslog.LOG_INFO, 'upssched-cmd "issuing shutdown"')
-            await run('/usr/local/sbin/upsmon', '-c', 'fsd', check=False)
+            # Before we start FSD with upsmon, lets ensure that ups is not ONLINE (OL).
+            # There are cases where battery/charger issues can result in ups.status being "OL LB" at the
+            # same time. This will ensure that we don't initiate a shutdown if ups is OL.
+            stats_output = (
+                await run(
+                    '/usr/local/bin/upsc', f'{config["identifier"]}@localhost:{config["remoteport"]}',
```
Comment (themylogin) [inline src/middlewared/middlewared/plugins/ups.py]: Does this work when we are in the slave mode? (I don't know)

File: src/middlewared/middlewared/plugins/ups.py (hunk 1 above)
Comment (william-gr) [reply src/middlewared/middlewared/plugins/ups.py]: We do not, we only use file logging.

If I am not mistaken that syslog was supposed to log to console. Weird that I am not seeing an `openlog`

**Comment by sonicaj** [comment]
backport

**Pull request merged**